src/containers/profile-page/components/desktop/SuggestedFollowsPopup.tsx

```diff
@@ -0,0 +1,167 @@
+import React, { RefObject, useCallback } from 'react'
+
+import { Popup, PopupPosition } from '@audius/stems'
+import cn from 'classnames'
+
+import { ReactComponent as IconClose } from 'assets/img/iconRemove.svg'
+import ArtistPopover from 'components/artist/ArtistPopover'
+import DynamicImage from 'components/dynamic-image/DynamicImage'
+import FollowButton from 'components/general/FollowButton'
+import { MountPlacement } from 'components/types'
+import { useUserProfilePicture } from 'hooks/useImageSize'
+import User from 'models/User'
+import { ID } from 'models/common/Identifiers'
+import { ProfilePictureSizes, SquareSizes } from 'models/common/ImageSizes'
+import zIndex from 'utils/zIndex'
+
+import styles from './SuggestedFollowsPopup.module.css'
+
+type SuggestedFollowsPopupProps = {
+  anchorRef: RefObject<HTMLElement>
+  artistName: string
+  suggestedArtists: Pick<
+    User,
+    | 'user_id'
+    | 'handle'
+    | 'name'
+    | '_profile_picture_sizes'
+    | 'does_current_user_follow'
+  >[]
+  isVisible: boolean
+  onClose: () => void
+  onArtistNameClicked: (handle: string) => void
+  onFollowAll: (userIds: ID[]) => void
+  onUnfollowAll: (userIds: ID[]) => void
+}
+
+const messages = {
+  follow: 'Follow All',
+  unfollow: 'Unfollow All',
+  following: 'Following'
+}
+
+export const SuggestedFollowsPopup = ({
+  anchorRef,
+  artistName,
+  suggestedArtists,
+  isVisible,
+  onClose,
+  onArtistNameClicked,
+  onFollowAll,
+  onUnfollowAll
+}: SuggestedFollowsPopupProps) => {
+  const onFollowAllClicked = useCallback(() => {
+    onFollowAll(suggestedArtists.map(a => a.user_id))
+  }, [suggestedArtists, onFollowAll])
+  const onUnfollowAllClicked = useCallback(() => {
+    onUnfollowAll(suggestedArtists.map(a => a.user_id))
+  }, [suggestedArtists, onUnfollowAll])
+
+  if (!suggestedArtists || suggestedArtists.length === 0) {
+    return null
+  }
+  return (
+    <Popup
+      position={PopupPosition.BOTTOM_LEFT}
+      // Popup should allow for non-mutable refs
+      // @ts-ignore
+      anchorRef={anchorRef}
```

Comment:
I think you should update the prop type to `MutableRefObject` because that's what `useRef` returns:

![image](https://user-images.githubusercontent.com/19916043/131906756-716e1062-4e12-424f-8451-a496d5a41b19.png)